code/drasil-data/Data/Drasil/Equations/Defining/Physics.hs

```diff
@@ -16,11 +16,12 @@ import Data.Drasil.Concepts.Documentation (body, constant)
 ------------------------------------------------------------------------------------------------------
 -- The equations
 
-newtonSLEqn, weightEqn, weightDerivAccelEqn, weightDerivNewtonEqn, weightDerivReplaceMassEqn,
+weightEqn, weightDerivAccelEqn, weightDerivNewtonEqn, weightDerivReplaceMassEqn,
   weightDerivSpecWeightEqn,
-  hsPressureEqn, speedEqn, velocityEqn, accelerationEqn  :: Relation
+  hsPressureEqn, speedEqn :: Relation
 
-newtonSLEqn               = sy QPP.mass `mulRe` sy QP.acceleration
+newtonSLEqn :: ModelExpr
+newtonSLEqn               = express $ sy QPP.mass `mulRe` sy QP.acceleration
```

Comment:
I initially changed it because it was only used in a TheoryModel. At the time, it only allowed ModelExprs in it, but I think we can loosen that requirement and revert this to an Expr.